core/src/main/java/brooklyn/util/crypto/FluentKeySigner.java

```diff
@@ -42,7 +44,9 @@
  * X509v3CertificateBuilder drags in an add'l dependency (bcmail) and is harder to use. */
 @SuppressWarnings("deprecation")
```

Comment:
I know it's not your code but couldn't help commenting...

I'd prefer us not to use a blanket `@SuppressWarnings("deprecation")` on the class. It means we don't get told what we're using that is actually deprecated, which we should fix (unless we're deliberately doing it for backwards compatibility reasons with multiple versions of a library?!).